lfucg-exactions/server/notes/viewsets.py

```diff
@@ -75,23 +78,45 @@ def create(self, request):
             return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
 
     def update(self, request, pk):
-        existing_object  = self.get_object()
-        setattr(existing_object, 'modified_by', request.user)
-        for key, value in request.data.items():
-            for existing_object_key, existing_object_value in existing_object.__dict__.items():
-                if key == existing_object_key:
-                    if value != existing_object_value:
-                        setattr(existing_object, existing_object_key, value)
-        try:
-            existing_object.save()
-            return Response(status=status.HTTP_200_OK)
-        except Exception as e:
-            return Response(status=status.HTTP_400_BAD_REQUEST)
+        self_table = request.data
+        self_table_is_active = self_table['is_active']
+
+        if self_table_is_active == True:
```

Comment:
use self_table['is_active'] as you're not using self_table_is_active anywhere else.